lib/ord.js

```diff
@@ -94,8 +94,14 @@ const _getPolicyLevel = (appConfig) =>
 const _getDescription = (appConfig) =>
     appConfig.env?.description || defaults.description;
 
-const _getProducts = (appConfig) =>
-    appConfig.env?.products || defaults.products(appConfig.packageName);
+const _getProducts = (appConfig) => {
```

Comment:
I think, we miss that we don't expose the product anymore, if the overwrite happened?!